kfdefs/overlays/moc/smaug/opf-dashboard/kfdef.yaml

```diff
@@ -0,0 +1,28 @@
+---
+apiVersion: kfdef.apps.kubeflow.org/v1
+kind: KfDef
+metadata:
+  annotations:
+    kfctl.kubeflow.io/force-delete: "false"
+  name: opendatahub
+spec:
+  applications:
+    - kustomizeConfig:
+        repoRef:
+          name: manifests
+          path: odh-dashboard
+      name: odh-dashboard
+    - kustomizeConfig:
+        overlays:
+          - custom-image
+          - cluster-role
+        repoRef:
+          name: opf
+          path: odh-manifests/smaug/dashboard
+      name: odh-dashboard
```

Comment:
Not sure if two kustomizeConfigs can have the same name, I just brought it in from the patch in the kustomization.yaml